README.md

```diff
@@ -24,6 +24,8 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 
 ### Unreleased
 
+* Modified Windows-All-IE11-1.15.xml
```

Comment:
Can you be a little more specific on what changed as a sub-bullet to this? Something like "Corrected registry key on V-46477"

Reply:
I've made those recommended changes.